### PR TITLE
Remove subscribe_to_campaign and cancel_campaign_subscription

### PR DESCRIPTION
**Pull request by RayeN**
They cannot be used by partners,

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,3 +1,9 @@
+DEV
+------------------
+* Remove `Yola.subscribe_to_campaign()` and `Yola.cancel_campaign_subscription`
+  mehtods, they aren't not allowed to be used externally.
```
Comment (highflyingaction) [inline CHANGELOG.rst]: Typo in `methods` and should just be `they aren't allowed`

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,3 +1,9 @@
+DEV
+------------------
+* Remove `Yola.subscribe_to_campaign()` and `Yola.cancel_campaign_subscription`
+  mehtods, they aren't allowed to be used externally.
```
Comment (highflyingaction) [inline CHANGELOG.rst]: Still a typo in `mehtods`.  Sorry, I should have made a list in the previous comment for clarity.

Comment (RayeN) [reply CHANGELOG.rst]: I'm just blind.  Sorry :)

**Review by highflyingaction — APPROVED** [review]

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Pull request merged**